react/src/App.jsx

```diff
@@ -27,12 +28,14 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/about-us" element={<AboutUs />} />
         <Route path="/calendar" element={<Calendar />} />
-        <Route path="/login" element={<Login onLogin={handleLogin} />} />
+        {/* <Route path="/login" element={<Login onLogin={handleLogin} />} /> */}
+        <Route path="/login" element={<Construction/>} />
         <Route path="/navigate" element={<Navigate />} />
-        <Route path="/register" element={<Register />} />
+        <Route path="/register" element={<Construction/>} />
         <Route path="/register-org" element={<RegisterOrg />} />
-        <Route path="/contact-us" element={<ContactUs />} />
+        <Route path="/contact-us" element={<Construction/>} />
```

Comment:
Should have kept them the same and just lined them out.  As you swapped out the button links in the NavBar. And added the `<Route path="/under-construction" element={<Construction/>} />`  as its own route. This will make it easier to turn it back on when its done.